students/Matt_Hudgins/Lesson09/mailroom_5.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+
+# chmod +x mailroom_5.py needs to be performed before executable
+
+'''
+    File Name: mailroom_5.py
+    Author: Matt Hudgins
+    Date created: 6/3/18
+    Date last modified: 6/3/18
+    Python Version 3.6.4
+'''
+import sys
+
+#Need Donor Class
+#Need Donor Collection Class
+#Interaction code should be outside the "model" class
+
+class Donor:
+
+    def __init__(self, name, donation=None):
+        self.donation = donation
+        self.name = name
+        self.total = sum(self.donation)
+        if donation is None:
+            self.donation = []
+        else:
+            self.donation = donation
+
+    @property
+    def add_donation(self, amount):
+        self.donation.append(amount)
+        self.total += amount
+        self.average = self.total / len(self.donation)
+
+class DonorDatabase:
+
+    def __init__(self, name=None):
+        self.donors = [] if name is None else name
+
+
+    def add_donor(self, name):
+        self.donors.append(name)
+
+
+    def get_donor(self, name):
```

Comment:
some notes about styling: 
* you should have 1 line space between class methods (module functions need 2 lines space)
* you also need 2 lines space between classes.